app/src/main/res/layout/player_seek_seconds_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Would it be possible to use a `ConstraintLayout` instead, so that there is less view nesting?

Reply:
Probably I could also try to use `<merge>` since it's a `LinearLayout` (I don't use constraints or anything similar) + extend from `LinearLayout`. I look into it, but I think it should work.